resources/assets/lib/beatmapsets-show/metadata-editor.tsx

```diff
@@ -81,6 +83,23 @@ export default class MetadataEditor extends React.PureComponent<Props, State> {
           </div>
         </label>
 
+        <label className='simple-form__row'>
+          <div className='simple-form__label'>
+            {osu.trans('beatmapsets.show.info.offset')}
+          </div>
+
+          <div className='form-text form-text--no-padding'>
+            <input
+              type='text'
+              name='beatmapset[offset]'
+              className='form-select__input'
```

Comment:
the classes make no sense. Use `simple-form__input` without wrapper and set its background color in the `--modal` modifier.